security/acme-client/src/opnsense/mvc/app/views/OPNsense/AcmeClient/actions.volt

```diff
@@ -48,6 +48,22 @@ POSSIBILITY OF SUCH DAMAGE.
             }
         );
 
+        // hook into on-show event for dialog to extend layout.
+        $('#DialogAction').on('shown.bs.modal', function (e) {
+            $("#action\\.type").change(function(){
+                var service_id = 'table_optional_' + $(this).val();
+                $(".table_optional").hide();
+                if (($("#action\\.type").val() == 'configd') || ($("#action\\.type").val() == 'custom')) {
```

Comment:
fixed in https://github.com/opnsense/plugins/pull/91/commits/6fcc7f8a5ae706279553601e606ee6d6da2b6766